docs/source/llms/langchain/index.rst

```diff
@@ -54,6 +54,14 @@ Supported Elements in MLflow LangChain Integration
 - `Agents <https://python.langchain.com/docs/modules/agents/>`_
 - `RetrievalQA <https://js.langchain.com/docs/modules/chains/popular/vector_db_qa>`_
 - `Retrievers <https://python.langchain.com/docs/modules/data_connection/retrievers/>`_
+- `Runnables <https://python.langchain.com/v0.1/docs/expression_language/interface/>`_
+- `LangGraph Complied Graph <https://langchain-ai.github.io/langgraph/reference/graphs/>`_ (only supported via `Model-from-Code <#logging-models-from-code>`_)
+
+
+.. warning::
+
+    There is a known deserialization issue when logging chain/agent that includes LangChain components from `the partner packages <https://python.langchain.com/v0.1/docs/integrations/platforms/#partner-packages>`_ such as ``langchain-openai``. If you log such models using the legacy serialization based logging, the some components might be loaded from ``langchain-community`` package instead of the partner package, which can lead to unexpected behavior.
```

Comment:
```suggestion
    There is a known deserialization issue when logging chains or agents dependent upon LangChain components from `the partner packages <https://python.langchain.com/v0.1/docs/integrations/platforms/#partner-packages>`_ such as ``langchain-openai``. If you log such models using the legacy serialization based logging, some components may be loaded from the respective ``langchain-community`` package instead of the partner package library, which can lead to unexpected behavior or import errors when executing your code.
```